lib/timeline/component/Group.js

```diff
@@ -475,6 +493,15 @@ Group.prototype.order = function() {
 Group.prototype._updateVisibleItems = function(orderedItems, oldVisibleItems, range) {
   var visibleItems = [];
   var visibleItemsLookup = {}; // we keep this to quickly look up if an item already exists in the list without using indexOf on visibleItems
+   
+  if (!this.isVisible) {
```

Comment:
I solved it by add equals in the 
`Group.prototype._isGroupVisible = function (range) {
   var isVisible = 
   (this.top <= range.body.domProps.centerContainer.height - range.body.domProps.scrollTop) 
   && (this.top + this.height >= - range.body.domProps.scrollTop);
   return isVisible;
 }`
So that when there are no groups, it will see it as one group with top 0 and we get 0==0 which leads to visible items.
This seems to solve the problem I caused. Examples now work well. Something doesn't sit well with me in this solution...

 think there should be a calculation of the initial group height and it should not be 0. It should calculate the height as if all the groups' items in the horizontal range are visible initially and then remove them if they aren't in the vertical range.
I'm working on a more elegant solution then the one supplied in this merge request.